pkgs/applications/graphics/sane/backends/brscan4/preload.c

```diff
@@ -0,0 +1,170 @@
+/* Brgen4 search for configuration under `/etc/opt/brother/scanner/brscan4`. This
```

Comment:
We seem to already have a generic LD_PRELOAD library `libredirect` - see e.g. `pkgs/applications/networking/instant-messengers/teamspeak/client.nix`  for examples.